ang/civicase/shared/directives/crm-editable.directive.js

```diff
@@ -28,7 +28,7 @@
             .html(
               textarea
                 ? nl2br(getHTMLToShow(scope, elem, attrs))
-                : _.escape(getHTMLToShow(scope, elem, attrs))
+                : getHTMLToShow(scope, elem, attrs)
```

Comment:
Please add unit tests.

Reply:
@deb1990 this whole file is missing unit tests. I'll confirm the priorities with @james-compucorp, but adding tests will take a long while.

Reply:
Okay in that case, if we can move this to a service(only the ternary operator part), can we just test that?

Reply:
Like a service to deal with Escaping/Unescaping strings.